### PR TITLE
Fix issue #5 shifting dimensions

### PR DESCRIPTION
**Pull request by RockingRishav**
Fix issue #5
fossweekend fix issue 5

### CODE REVIEW

**Comment by RockingRishav** [comment]
Checks have failed what is the issue with this PR

**Comment by Rohit1173** [comment]
Replace `dimens` with `dimen` everywhere

**Comment by Rohit1173** [comment]
> Fix issue#5

give space in between issue and #

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -1,2 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources></resources>
\ No newline at end of file
+<resources>
+    <dimens name="dmn_wid_hig">125dp</dimens>
+    <dimens name="dmn_margin_top">110dp</dimens>
+    <dimens name="dmn_padding">0dp</dimens>
+    <dimens name="dmn_margin_tp">75dp</dimens>
+    <dimens name="dmn_margin_lr">20dp</dimens>
+    <dimens name="dmn_margin_tpe">40dp</dimens>
```
Comment (ayush0402) [inline app/src/main/res/values/dimens.xml:8]: 'dimens' is not a valid tag type. Replace every 'dimens' with `dimen`. Checks should pass then.

**Review by ayush0402 — CHANGES_REQUESTED** [review]
@RockingRishav Requested some changes PTAL.

**Comment by Rohit1173** [comment]
@RockingRishav the naming could be done better like `@dimen/msgText_marginTop`, use id of the view you are using in naming the `dimen` so that we could differentiate them easily.

**Comment by Rohit1173** [comment]
LGTM,Thanks.

**Pull request merged**